feature_engine/output/jury_TINY_output_conversation_level.csv

```diff
@@ -1,3 +1,3 @@
-,batch_num,round_num,gini_coefficient_num_words,gini_coefficient_num_chars,average_message_count,most_talkative_member_message_count,least_talkative_member_message_count,std_message_count,average_word_count,most_talkative_member_word_count,least_talkative_member_word_count,std_word_count
-0,0,0,0.15825554259043173,0.1685722411831627,6.5,9,1,2.3979157616563596,107.125,140,31,33.6654032353691
-1,0,2,0.12569549822964088,0.1255592841163311,7.5,12,3,3.685557397915997,109.83333333333333,169,76,28.457082695799222
+batch_num,round_num,gini_coefficient_num_words,gini_coefficient_num_chars,average_message_count_x,average_message_count_y,average_word_count,std_word_count,max_messages,min_messages,max_words,min_words
```

Comment:
Noticing that average_message_count is showing up twice in the output .... did we add that column twice in merge somehow?